src/ia-anniversary-banner.ts

```diff
@@ -0,0 +1,194 @@
+import { html, css, LitElement, property, customElement } from 'lit-element';
+import { nothing } from 'lit-html';
+import CloseCircleIcon from '@internetarchive/icon-close-circle';
+
+import { IAFile, IAMD, TimelineMoment } from './interfaces';
+
+import { formatMoments } from './utils';
+import wayforwardSvg from './images/way-forward-svg';
+import wayforwardSvgMin from './images/way-forward-min-svg';
+
+import './marquee';
+
+@customElement('ia-anniversary-banner')
+export class IaAnniversaryBanner extends LitElement {
+  @property({ type: Object }) iaFiles: IAFile[] = [];
+
+  @property({ type: Object }) iaMD: IAMD = { directory: '', separator_dir: '' };
+
+  @property({ type: Object }) moments: { [key: string]: TimelineMoment } = {};
+
+  @property({ type: Array }) shuffledMoments: TimelineMoment[] = [];
+
+  @property({ type: String }) landingURL = 'https://archive.org';
+
+  @property({ type: String }) viewMode = 'open'; // open | closed
```

Comment:
Question and a suggestion:
1. If the banner is closed, should the consumer of the banner just remove it from the DOM (vs setting this to closed to not render it)
2. If the banner is going to be left in the DOM, I would use a `type` or `enum` like the dev menu [suggestion above](https://github.com/internetarchive/iaux-25th-anniversary-banner/pull/1/files#r664721974).